Note sample workflows in `commands.rst` (#149)
python-docs-translations/transifex-automations: docs/commands.rst

This document contains instructions to guide you on managing your language
repository, hence helping you translate Python's documentation to your language.

> The :doc:`sample workflow's <workflows>` Transifex workflow is an automated
> alternative to this guide.

Reviewer: Sounds weird to me "The sample workflow's Transifex workflow ". How about (were 'workflows' ref is replaced with GitHub Worfklow):

```suggestion
See :doc:`workflows`, an automated alternative to this guide in GitHub Actions.
```
